### PR TITLE
tests: Add example test for react component

### PR DESCRIPTION
**Pull request by Dignifiedquire**
cc @RichardLitt @diasdavid

### CODE REVIEW

File: app/scripts/views/config.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default React.createClass({
 
   getInitialState: function () {
     return {
-      body: JSON.stringify(this.props.config, null, '  '),
+      body: JSON.stringify(this.props.config, null, 2),
```
Comment (travisperson) [inline app/scripts/views/config.js]: This just blew my mind a little bit, never knew you could specify a number instead of a string.

Comment (Dignifiedquire) [inline app/scripts/views/config.js]: :)

File: package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "karma-mocha": "^0.2.1",
     "karma-sourcemap-loader": "^0.3.6",
     "karma-webpack": "^1.7.0",
+    "legit-tests": "^0.6.3",
```
Comment (diasdavid) [inline package.json]: so legit!

**Comment by diasdavid** [comment]
This looks nice! Can we also have a test example for a piece of functionality?

**Comment by Dignifiedquire** [comment]
@diasdavid what kind of functionality do you mean?

**Comment by diasdavid** [comment]
Right now the test is to see if it the view is rendered correctly, right? I guess that every test can be performed by an amount of assertions on the elements that were rendered, but for things like "Check if the files uploaded list is correct" could be tested without the UI part?

**Comment by Dignifiedquire** [comment]
Closing in favor of #156

**Pull request closed without merging**